### DISCUSSION TITLE
Supervised Learning

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Mathematical-Foundations-of-ML#23] [Q&A] [answered]
Which method we use to divide a dataset into train and test set in python? write the syntax.

**Comment by Yash-slr** [comment, marked as answer]
For dividing the dataset in python we use "train_test_split" method from "sklearn.model_selection".                                                                                Syntax is :-                                                                                                                                                                                                                           let us take "X" as independent variables and "y" as dependent variables                                                                                                                                                                                                                    from sklearn.model_selection import train_test_split
X_train, X_test, y_train, y_test = train_test_split(arrays, test_size=___, random_state=___)                                                                                                    
                                                                                                                                                                                                                             
    Any changes to be made mam to for better answer?
